cypress/e2e/cloud/home.test.ts

```diff
@@ -39,7 +43,7 @@ describe('Home Page Tests', () => {
 ,,0,2021-08-05T20:54:31.642101603Z,Alpha,07f205512a38f000,Alpha,ok
 ,,0,2021-08-05T20:54:00Z,Check: Beta is: ok,07f2055f7a12d000,Beta,ok`
 
-    cy.intercept('POST', '/api/v2/query?*', req => {
+    cy.intercept('POST', `/api/v2/query?orgID=${orgID}`, req => {
```

Comment:
the crux of the migration is twofold:

1. Intercepts must now abide by strict path matching, no more wildcards
2. Intercepts now go in the the opposite order they were declared for .... reasons (check the docs)